SYCL/ESIMD/fp_call_from_func.cpp

```diff
@@ -7,7 +7,7 @@
 //===----------------------------------------------------------------------===//
 // REQUIRES: gpu
 // Issue #162 Test timeouts on Windows and Linux
```

Comment:
```suggestion
// Issue #163 Test timeouts on Windows and Linux
```
@vladimirlaz, it looks like the right issue number is 163, not 162. Am I right?

Reply:
you are right